<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFieldPreferences</class>
 <widget class="QWidget" name="QFieldPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFieldSync Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="cloudGroupBox">
     <property name="title">
      <string>QFieldCloud Settings</string>
     </property>
     <layout class="QGridLayout" name="cloudGridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="cloudDirectoryLabel">
        <property name="text">
         <string>Default cloud directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="cloudIntroLabel">
        <property name="text">
         <string>&lt;span style=&quot; font-style:italic;&quot;&gt;&lt;a href=&quot;https://qfield.cloud/&quot;&gt;QFieldCloud&lt;/a&gt; allows your team to focus on what's important, making sure you efficiently get quality field data. Thanks to its integration with QField, you and your team will be able to start surveying and digitising data collaboratively in no time.&lt;/span&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsFileWidget" name="cloudDirectory"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="qfieldCloudRememberMe">
        <property name="text">
         <string>Stay signed in</string>
        </property>
        <property name="toolTip">
         <string>Store QFieldCloud credentials and automatically sign in on QGIS startup.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="generalGroupBox">
     <property name="title">
      <string>Packaging Settings</string>
     </property>
     <layout class="QGridLayout" name="generalGridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="cloudIntroLabel">
        <property name="text">
         <string>&lt;span style=&quot; font-style:italic;&quot;&gt;QField continues to give users full control by allowing projects to be packaged and synchronized via USB cable as well as through networks.&lt;/span&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="importDirectoryLabel">
        <property name="text">
         <string>Default packaging import directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsFileWidget" name="importDirectory"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="exportDirectoryLabel">
        <property name="text">
         <string>Default packaging export directory</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QgsFileWidget" name="exportDirectory"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="showPackagingActions">
        <property name="text">
         <string>Show packaging actions in the toolbar</string>
        </property>
        <property name="toolTip">
         <string>Store QFieldCloud credentials and automatically sign in on QGIS startup.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>56</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
